src/Parts/StickerPack.php

```diff
@@ -8,7 +8,7 @@ class StickerPack
 {
     public string $id;
     /**
-     * @var \Ragnarok\Fenrir\Parts\Sticker[]
+     * @var Sticker[]
```

Comment:
3rd party json mapper requires complete paths in doc blocks (not blaming you for not knowing about this limitation)